signal_scanner_bot/env.py

```diff
@@ -60,6 +96,10 @@ def _cast_to_int(to_cast: str) -> int:
     return int(to_cast)
 
 
+def _cast_to_path(to_cast: str) -> Path:
+    return Path(to_cast)
```

Comment:
Your call here since this is consistent with our other conversion functions, but we could remove this function and just pass `Path` as the `convert` keyword directly in the `_env` call (e.g. `_env(..., convert=Path)`)

Reply:
I wanted to do a cast function here to allow us to easily add verifications/checks (e.g. like make sure the path provided doesn't have a leading `/` or `~` because we want the file to always be in the `signal_scanner_bot` directory otherwise the state file won't persist) but didn't want to actually write them right now because I couldn't think of any actually good ones and was just gonna make an issue to think about it later.

Reply:
Got it, totally makes sense! :+1: